CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+Code of Conduct
+---------------
+
+At webpack and all webpack/webpack-contrib repositories we follow the
```

Comment:
Maybe I think it should be specific to `webpack-cli` only as this is for `webpack-cli`.